src/iqm/cirq_iqm/devices/aphrodite.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2020–2025 Cirq on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+IQM's Aphrodite quantum architecture.
+"""
+from __future__ import annotations
+
+from .iqm_device import IQMDevice, IQMDeviceMetadata
+
+
+class Aphrodite(IQMDevice):
+    r"""IQM's twenty-qubit transmon device.
+
+    The qubits are connected thus::
+
+            QB54  QB51  QB46  QB39
+            /  \  /  \  /  \  /  \ 
+        QB53  QB50  QB45  QB38  QB31
+        /  \  /  \  /  \  /  \  / 
+        QB52  QB49  QB44  QB37  QB30
+        \  /  \  /  \  /  \  /  \ 
+        QB48  QB43  QB36  QB29  QB22
+        /  \  /  \  /  \  /  \  /  \ 
+        QB47  QB42  QB35  QB28  QB21  QB14
+        \  /  \  /  \  /  \  /  \  / 
+        QB41  QB34  QB27  QB20  QB13
+        /  \  /  \  /  \  /  \  /  \ 
+        QB40  QB33  QB26  QB19  QB12  QB7
+        \  /  \  /  \  /  \  /  \  / 
+        QB32  QB25  QB18  QB11  QB6
+            \  /  \  /  \  /  \  /  \ 
+            QB24  QB17  QB10  QB5   QB2
+            /  \  /  \  /  \  /  \  / 
+        QB23  QB16  QB9   QB4   QB1
+            \  /  \  /  \  / 
+            QB15  QB8   QB3
```

Comment:
Some connections are misaligned, this would be better:
```suggestion
          QB54  QB51  QB46  QB39
          /  \  /  \  /  \  /  \
       QB53  QB50  QB45  QB38  QB31
       /  \  /  \  /  \  /  \  /
    QB52  QB49  QB44  QB37  QB30
       \  /  \  /  \  /  \  /  \
       QB48  QB43  QB36  QB29  QB22
       /  \  /  \  /  \  /  \  /  \
    QB47  QB42  QB35  QB28  QB21  QB14
       \  /  \  /  \  /  \  /  \  /
       QB41  QB34  QB27  QB20  QB13
       /  \  /  \  /  \  /  \  /  \
    QB40  QB33  QB26  QB19  QB12  QB7
       \  /  \  /  \  /  \  /  \  /
       QB32  QB25  QB18  QB11  QB6
          \  /  \  /  \  /  \  /  \
          QB24  QB17  QB10  QB5   QB2
          /  \  /  \  /  \  /  \  /
       QB23  QB16  QB9   QB4   QB1
          \  /  \  /  \  /
          QB15  QB8   QB3
```

Reply:
One more of these and it'll be quicker to write a script which produces the image :)